dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/VelocitySessionWrapper.java

```diff
@@ -112,5 +112,35 @@ public void setMaxInactiveInterval(int arg0) {
 
     }
 
+    @Override
+    public int hashCode() {
+        final int prime = 31;
+        int result = 1;
+        result = prime * result + ((session == null) ? 0 : session.hashCode());
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'obj' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5378571)